packages-web/fieldset/src/Fieldset.editorPreview.tsx

```diff
@@ -0,0 +1,32 @@
+import { createElement, ReactElement, ComponentType } from "react";
+
+import { Fieldset } from "./components/Fieldset";
+import { FieldsetPreviewProps } from "../typings/FieldsetProps";
+
+declare function require(name: string): string;
+
+// Temporary interface to fix incorrect widgets property typing for preview ("renderer: ReactNode" should be "renderer: ComponentType")
```

Comment:
The new properties are already in place, you should just remove node_modules and package-lock ann `npm install`

Reply:
Better wait for #172 to land and rebase. Then you can do ☝️

Reply:
Changed preview typings.